mint/fakeAPIData.ts

```diff
@@ -284,12 +325,51 @@ export const prevGraph: GraphDefinition = {
     }
 };
 
+export const costGraph: GraphDefinition = {
+    metadata: {
+        x_col: "cases_averted",
+        id_col: "intervention",
+        format: "long",
+        settings: ["irs_use", "net_use"]
+    },
+    series: [
+        {
+            id: "ITN",
+            y_formula: ["({population} / {procure_people_per_net}) * {price_net_standard}  + ({price_delivery} * {population} * (({procure_buffer} + 100)/ 100))"],
```

Comment:
nb - we've changed over to using camelCase in mintr for the data, so should probably switch over the graph components to do the same?